docs/node/installation.md

```diff
@@ -101,7 +103,10 @@ sudo apt-get install \
     libssl-dev \
     libusb-1.0-0-dev \
     libzmq3-dev \
-    uuid-dev
+    uuid-dev \
+    libre2-dev \
+	libglib2.0-0 libglib2.0-dev \
```

Comment:
```suggestion
	libglib2.0-dev \
```

Please note, that its only required to list development packages here.
These packages have the library packages as a dependency as shown at the following link under the "Requires" section:

https://debian.pkgs.org/12/debian-main-amd64/libglib2.0-dev_2.74.6-2+deb12u5_amd64.deb.html#Requires